Added information to Logpush retry logic. (#3349)
cloudflare/cloudflare-docs: products/logs/src/content/faq/index.md

### What happens if my cloud storage destination is temporarily unavailable?

> **Logpush** is designed to retry in case of errors. If your destination is temporarily unavailable, Logpush will retry around five times over five minutes, but be aware that this number and time are just approximations. If Cloudflare persistently receives errors from your destination, and cannot keep up with incoming batches, Logpush will eventually drop logs. If the errors continue for a prolonged period of time, Logpush will assume that the destination is permanently unavailable and disable your push job. You can always re-enable the job later.

Reviewer: ```suggestion
**Logpush** is designed to retry in case of errors. If your destination is temporarily unavailable, Logpush will retry around five times over five minutes. However, note that this number and time are just approximations. If Cloudflare persistently receives errors from your destination, and cannot keep up with incoming batches, Logpush will eventually drop logs. If the errors continue for a prolonged period of time, Logpush will assume that the destination is permanently unavailable and disable your push job. You can always re-enable the job later.
```

Reviewer: Just a suggestion. The sentence was long, and it reads better if we break it into 2. Also, I personally prefer to say "note" than "be aware". But I leave this second part of the suggestion up to you :)
